### PR TITLE
fix(addFriends): click Add Friends to swipe into friends page

### PR DESCRIPTION
**Pull request by nickjjang**
AP-371

### CODE REVIEW

File: components/views/navigation/sidebar/Sidebar.vue [hunk 1]
```diff
@@ -65,17 +65,15 @@ export default Vue.extend({
...
-    }
+    // if (this.$route.name?.includes('chat-direct') && this.$device.isMobile) {
+    //   this.$props.showMenu()
+    // }
+    // if (this.$route.name?.includes('friends-list') && this.$device.isMobile) {
+    //   this.$props.showMenu()
+    // }
+    // if (this.$route.name?.includes('files-browse') && this.$device.isMobile) {
+    //   this.$props.showMenu()
+    // }
```
Comment (stavares843) [inline components/views/navigation/sidebar/Sidebar.vue:76]: commented-out code 🔨

File: layouts/chat.vue [hunk 2]
```diff
@@ -160,14 +175,16 @@ export default Vue.extend({
     }
     window.addEventListener('resize', appHeight)
     appHeight()
+
+    if (this.$device.isMobile && this.$route.params.id) {
+      this.$store.commit('ui/showSidebar', false)
+    } else {
```
Comment (WanderingHogan) [inline layouts/chat.vue:181]: can you do a guard clause here

File: layouts/files.vue [hunk 3]
```diff
@@ -68,33 +66,50 @@ export default Vue.extend({
...
+        newValue
+          ? this.$refs.swiper.$swiper.slidePrev()
+          : this.$refs.swiper.$swiper.slideNext()
       }
+    },
+  },
+  mounted() {
+    if (this.$device.isMobile && !this.$route.query?.sidebar) {
+      this.$store.commit('ui/showSidebar', false)
+    } else {
```
Comment (WanderingHogan) [inline layouts/files.vue:104]: guard clause instead of else

File: layouts/friends.vue [hunk 4]
```diff
@@ -67,32 +65,49 @@ export default Vue.extend({
...
     }
   },
   computed: {
     ...mapState(['friends']),
+    ...mapGetters('ui', ['showSidebar']),
+  },
+  mounted() {
+    if (this.$device.isMobile && !this.$route.query?.sidebar) {
+      this.$store.commit('ui/showSidebar', false)
+    } else {
```
Comment (WanderingHogan) [inline layouts/friends.vue:94]: guard

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
couple small changes,

does the sidebar state update make the other sidebar state settings obsolete? or are they still used somewhere?

File: layouts/files.vue (hunk 3 above)
Comment (nickjjang) [reply layouts/files.vue:104]: Oh, I got guard clause thanks. and I've fixed.

File: components/views/navigation/sidebar/Sidebar.vue (hunk 1 above)
Comment (nickjjang) [reply components/views/navigation/sidebar/Sidebar.vue]: Resolved

File: layouts/chat.vue (hunk 2 above)
Comment (nickjjang) [reply layouts/chat.vue]: Resolved

File: layouts/files.vue (hunk 3 above)
Comment (nickjjang) [reply layouts/files.vue]: Resolved

File: layouts/friends.vue (hunk 4 above)
Comment (nickjjang) [reply layouts/friends.vue]: Resolved

**Review by WanderingHogan — APPROVED** [review]

**Comment by stavares843** [comment]
Captura de ecrã 2022-01-07, às 20 14 08

fixed

added a separate ticket for the side menu overlap icon - [AP-426](https://satellite-im.atlassian.net/browse/AP-426)

**Review by stavares843 — APPROVED** [review]

**Pull request merged**